CONTRIBUTING.md

```diff
@@ -1,15 +1,15 @@
-OBS.Ninja Contributor License Agreement (CLA)
+# OBS.Ninja Contributor License Agreement (CLA)
 
 To ensure the long-term viability of the open-source OBS.Ninja project, and for the protection of its creator and its users, we request that contributors to the project first agree to some basic terms. The terms when accepted applies to all of your past, present and future contributions.
 
-Contribution Policy
+## Contribution Policy

```

Comment:
I'd make this the same level as the previous header, not a lower level. So, # instead of ##.